frontend/packages/console-shared/src/components/drawer/drawer-utils.ts

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
```

Comment:
I don't see a need for any of these `drawer-utils` if you use `DraggableCore` correctly and fix the size of the header according to the component design.
But if you want to still get the fixed height of the header at runtime using an effect with the ref on the header wrapper. But since we only care about changes in height, simplify the hook and only notify on height change.